pyseir/__init__.py

```diff
@@ -0,0 +1,3 @@
+import os
+DATA_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'data')
+OUTPUT_DIR = os.path.join(os.path.dirname(os.path.abspath(__file__)), '..', 'output')
```

Comment:
If you wanted to use the pathlib interface, I think it's cleaner! (and I love pathlib and take any chance I get to evangelize it).

```suggestion
import pathlib
DATA_DIR = pathlib.Path(__file__).parent.parent / 'data'
OUTPUT_DIR = pathlib.Path(__file__).parent.parent / 'output'
```